markhor_bringup/package.xml

```diff
@@ -12,5 +12,6 @@
 
```

Comment:
I think we will need the  `build_depend`  tag also. See this page http://wiki.ros.org/ROS/Tutorials/rosdep